gemfiles/graphql_2.0.gemfile

```diff
@@ -2,6 +2,6 @@
 
 source "https://rubygems.org"
 
-gem "graphql", "~> 2.0"
+gem "graphql", "2.0.19"
```

Comment:
shouldn't be here something like `~> 2.0.19` to keep bugfixes versions compatible 🤔

Reply:
This is just for the CI. It's not going to block upgrading the dependency on the application side.
Making it too loose might cause that the CI will be unstable when a new version gets released.
If someone wants to upgrade the development dependencies I think it's better if they do it consciously IMO.